package.json

```diff
@@ -38,6 +38,9 @@
 		"promises",
 		"bluebird"
 	],
+	"dependencies": {
+		"random-int": "^2.0.1"
```

Comment:
`delay` is a very popular package, so I would like to keep it dependency-free. Can you just inline https://github.com/sindresorhus/random-int/blob/c37741b56f76b9160b0b63dae4e9c64875128146/index.js#L13-L15 ?

Reply:
Fixed!